mzLib.sln

```diff
@@ -33,103 +33,267 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test", "Test\Test.csproj",
+		{0DE04F7B-EE40-4055-ACF7-472167B70A04}.TestAndRelease|x86.Build.0 = Release|x64
+		{244A891E-BCDA-432F-8160-26EE0C5F8096}.Debug|Any CPU.ActiveCfg = Debug|x64
 		{244A891E-BCDA-432F-8160-26EE0C5F8096}.Debug|x64.ActiveCfg = Debug|x64
 		{244A891E-BCDA-432F-8160-26EE0C5F8096}.Debug|x64.Build.0 = Debug|x64
+		{244A891E-BCDA-432F-8160-26EE0C5F8096}.Debug|x86.ActiveCfg = Debug|x64
+		{244A891E-BCDA-432F-8160-26EE0C5F8096}.Release|Any CPU.ActiveCfg = Release|x64
 		{244A891E-BCDA-432F-8160-26EE0C5F8096}.Release|x64.ActiveCfg = Release|x64
 		{244A891E-BCDA-432F-8160-26EE0C5F8096}.Release|x64.Build.0 = Release|x64
+		{244A891E-BCDA-432F-8160-26EE0C5F8096}.Release|x86.ActiveCfg = Release|x64
+		{244A891E-BCDA-432F-8160-26EE0C5F8096}.TestAndRelease|Any CPU.ActiveCfg = Release|x64
+		{244A891E-BCDA-432F-8160-26EE0C5F8096}.TestAndRelease|Any CPU.Build.0 = Release|x64
 		{244A891E-BCDA-432F-8160-26EE0C5F8096}.TestAndRelease|x64.ActiveCfg = Release|x64
 		{244A891E-BCDA-432F-8160-26EE0C5F8096}.TestAndRelease|x64.Build.0 = Release|x64
+		{244A891E-BCDA-432F-8160-26EE0C5F8096}.TestAndRelease|x86.ActiveCfg = Release|x64
+		{244A891E-BCDA-432F-8160-26EE0C5F8096}.TestAndRelease|x86.Build.0 = Release|x64
+		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.Debug|Any CPU.ActiveCfg = Debug|x64
 		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.Debug|x64.ActiveCfg = Debug|x64
 		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.Debug|x64.Build.0 = Debug|x64
+		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.Debug|x86.ActiveCfg = Debug|x64
+		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.Release|Any CPU.ActiveCfg = Release|x64
 		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.Release|x64.ActiveCfg = Release|x64
 		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.Release|x64.Build.0 = Release|x64
+		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.Release|x86.ActiveCfg = Release|x64
+		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.TestAndRelease|Any CPU.ActiveCfg = Release|x64
+		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.TestAndRelease|Any CPU.Build.0 = Release|x64
 		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.TestAndRelease|x64.ActiveCfg = Release|x64
 		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.TestAndRelease|x64.Build.0 = Release|x64
+		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.TestAndRelease|x86.ActiveCfg = Release|x64
+		{707DB7D7-3C7F-4E2C-91D9-A933D4FCAE2F}.TestAndRelease|x86.Build.0 = Release|x64
+		{D2B8F696-1172-405B-B788-3251B27CC71A}.Debug|Any CPU.ActiveCfg = Debug|x64
 		{D2B8F696-1172-405B-B788-3251B27CC71A}.Debug|x64.ActiveCfg = Debug|x64
 		{D2B8F696-1172-405B-B788-3251B27CC71A}.Debug|x64.Build.0 = Debug|x64
+		{D2B8F696-1172-405B-B788-3251B27CC71A}.Debug|x86.ActiveCfg = Debug|x64
+		{D2B8F696-1172-405B-B788-3251B27CC71A}.Release|Any CPU.ActiveCfg = Release|x64
 		{D2B8F696-1172-405B-B788-3251B27CC71A}.Release|x64.ActiveCfg = Release|x64
 		{D2B8F696-1172-405B-B788-3251B27CC71A}.Release|x64.Build.0 = Release|x64
+		{D2B8F696-1172-405B-B788-3251B27CC71A}.Release|x86.ActiveCfg = Release|x64
```

Comment:
Do you need these x86 and AnyCPU configurations? I think x64 is the only one that's used currently to allow > 2 GB RAM usage, so consider removing these.